cmd/certsuite/run/run.go

```diff
@@ -34,7 +34,7 @@ func NewCommand() *cobra.Command {
 	runCmd.PersistentFlags().String("log-level", "debug", "Sets the log level")
 	runCmd.PersistentFlags().String("offline-db", "", "Set the location of an offline DB to check the certification status of for container images, operators and helm charts")
 	runCmd.PersistentFlags().String("preflight-dockerconfig", "", "Set the dockerconfig file to be used by the Preflight test suite")
-	runCmd.PersistentFlags().Bool("non-intrusive", false, "Run only the test that do not disrupt the test environment")
+	runCmd.PersistentFlags().Bool("non-intrusive", false, "Run intrusive tests that may disrupt the test environment")
```

Comment:
Why the comment change? Was this incorrect in the first place? Or is this a flag where it's truthiness does the opposite of what the flag states?

Reply:
I believe it was incorrect in the first place.
According to [doc](https://redhat-best-practices-for-k8s.github.io/certsuite/runtime-env/#ocp-412-labels), to enable intrusive tests, export CERTSUITE_NON_INTRUSIVE_ONLY=false.

Reply:
Maybe I'm reading this incorrectly, but it seems the flags name, it's truthiness, or it's description are incorrect. 

@sebrandon1 Am I just reading this incorrectly?

Reply:
I know that this increases the scope of this ask, but if we can fix this confusion now and switch to `--intrusive: true` as the default and globally refactor to drop the `non` elsewhere, I'm all for that to avoid confusion in the future.

Reply:
Agreed, let's just adjust in one shot.

Reply:
I will refactor shortly.

Reply:
@sebrandon1 Do you know what is the DISABLE_INTRUSIVE_TESTS in the workflows set to?
I'm not so sure how is this env related to --intrusive.
- name: Run the tests (against binary)
        uses: nick-fields/retry@ce71cc2ab81d554ebbe88c79ab5975992d79ba08 # v3.0.2
        with:
          timeout_minutes: 60
          max_attempts: 3
          command: cd ${GITHUB_WORKSPACE}/certsuite-qe; FEATURES=${{matrix.suite}} CERTSUITE_REPO_PATH=${GITHUB_WORKSPACE} USE_BINARY=true JOB_ID=${{github.run_id}} DISABLE_INTRUSIVE_TESTS=false ENABLE_PARALLEL=true ENABLE_FLAKY_RETRY=true make test-features

Reply:
So let's just leave those, because those flags are for the [certsuite-qe](https://github.com/redhat-best-practices-for-k8s/certsuite-qe) repo and they just cause certain tests over there to be skipped completely.

The flag is here for reference: https://github.com/redhat-best-practices-for-k8s/certsuite-qe/blob/8864991067367859352ecddfc22498b00a8374b9/tests/utils/config/config.go#L39